ui/shared/utils/panelAppUtils.js

```diff
@@ -0,0 +1,51 @@
+export const moiToMoiTypes = (rawMoi) => {
```

Comment:
Yeah, especially since we're not even showing initials for all of them.  This PR will already provide a reference for what each raw mode_of_inheritance value means (which is somewhat arbitrary in the first place) and we'll refer to it if we want more initials in the future.

Reply:
Ah yes, the MOI Type was originally used because I was expecting to put in Symbols or Emojis instead of initials. And then when they weren't all being shown, I thought it would be more accurately testable with the MOI Type & more future proof if we were to show the MOI Type elsewhere.

But yes, it is an uneccessary step now and I'll remove it for a moiToMoiInitials function.